### PR TITLE
feat: Add kms-solutions repo into CFT

### PR DESCRIPTION
**Pull request by romanini-ciandt**
The goal of this PR is to link CFT with [kms-solutions repo](https://github.com/GoogleCloudPlatform/kms-solutions)

### CODE REVIEW

**Review by tdbhacks — APPROVED** [review]
LGTM!

File: infra/terraform/test-org/org/locals.tf [hunk 1]
```diff
@@ -875,6 +875,16 @@ locals {
       lint_env = {
         ENABLE_BPMETADATA = "1"
       }
+    },
+    {
+      name            = "kms-solutions"
+      org             = "GoogleCloudPlatform"
+      description     = "Store Cloud KMS scripts, artifacts, code samples, and more."
+      owners          = ["tdbhacks"]
+      enable_periodic = true
```
Comment (bharathkkb) [inline infra/terraform/test-org/org/locals.tf:884]: Do we need periodics since not JSS?

**Review by bharathkkb — APPROVED** [review]

File: infra/terraform/test-org/org/locals.tf [hunk 2]
```diff
@@ -875,6 +875,16 @@ locals {
       lint_env = {
         ENABLE_BPMETADATA = "1"
       }
+    },
+    {
+      name            = "kms-solutions"
```
Comment (g-awmalik) [inline infra/terraform/test-org/org/locals.tf:880]: Is this different from https://github.com/terraform-google-modules/terraform-google-kms?

Comment (apeabody) [reply infra/terraform/test-org/org/locals.tf:880]: Yeah - It's not a true "tf module", but this repo: https://github.com/GoogleCloudPlatform/kms-solutions

Comment (tdbhacks) [reply infra/terraform/test-org/org/locals.tf:880]: Indeed, the one you linked is the terraform KMS module, whereas `kms-solutions` is a new repo where we plan to store samples, artifacts, tutorials, and some terraform blueprints that might not fit the KMS module repo.

File: infra/terraform/test-org/org/locals.tf (hunk 1 above)
Comment (apeabody) [reply infra/terraform/test-org/org/locals.tf:884]: @romanini-ciandt?

That said, I don't think it should cause any issues.

**Comment by apeabody** [comment]
Ideally https://github.com/GoogleCloudPlatform/kms-solutions/pull/2 will be merged first

File: infra/terraform/test-org/org/locals.tf (hunk 1 above)
Comment (romanini-ciandt) [reply infra/terraform/test-org/org/locals.tf]: Nop, I don't think we need. It's removed!

Comment (apeabody) [reply infra/terraform/test-org/org/locals.tf]: Thanks @romanini-ciandt - Can you please do a `terraform fmt` on the file.

Comment (apeabody) [reply infra/terraform/test-org/org/locals.tf]: I took care of it.

**Comment by apeabody** [comment]
applied

File: infra/terraform/test-org/org/locals.tf (hunk 1 above)
Comment (romanini-ciandt) [reply infra/terraform/test-org/org/locals.tf]: Thank you @apeabody

**Pull request merged**